Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/CovidTreatmentAssessment.vue

```diff
@@ -151,7 +151,7 @@ export default class CovidTreatmentAssessmentComponent extends Vue {
     }
 
     private get patientFullName(): string {
-        return `${this.patient.firstname} ${this.patient.lastname} `;
+        return `${this.patient.preferredName.givenName} ${this.patient.preferredName.surname} `;
```

Comment:
This is AdminWebClient and it calls v2 so I don't think there should be any changes to this code.